lib/nxdk/xbe.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Stefan Schmidt
+ *
+ * Licensed under the MIT License
+ */
+
+#ifndef __NXDK_XBE_H__
+#define __NXDK_XBE_H__
+
+#ifdef __cplusplus
+{
+#endif
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+void XGetXbePath (char *path, bool includeFileName);
```

Comment:
My original idea for moving this into a more generic `paths.h` was to make this more extensible.
There's other standard parts the user might want to query like `XGetSavedataPath(..., uint32_t titleId);` or `XGetDashboardXbePath(...)`.

The path of the current XBE also isn't really a part of the XBE itself. I'd prefer to reserve the `xbe.h` name for functions and types that actually deal with XBE images.

Reply:
I wasn't really sure where to put it, but yeah, `path.h` makes more sense.